openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamHierarchy.tsx

```diff
@@ -90,8 +91,14 @@ const TeamHierarchy: FC<TeamHierarchyProps> = ({
       {
         title: t('label.description'),
         dataIndex: 'description',
+        width: 450,
         key: 'description',
-        render: (description: string) => description || '--',
+        render: (description: string) =>
+          !isEmpty(description) ? (
+            <RichTextEditorPreviewer markdown={description} />
+          ) : (
+            '--'
+          ),
```

Comment:
let's just show 2 line of the description and an end ellipsis, you can refer to this https://github.com/open-metadata/OpenMetadata/blob/55110f93c48cc9f00542f55f787e9d36e1946291/openmetadata-ui/src/main/resources/ui/src/components/Glossary/ImportResult/ImportResult.tsx#L119